src/signals/incident-management/containers/IncidentDetail/IncidentDetail.tsx

```diff
@@ -71,24 +75,44 @@ const Preview = styled.div`
 const IncidentDetail = () => {
   const { emit, listenFor, unlisten } = useEventEmitter()
```

Comment:
er moet ook nog een unlisten in de gereturnde functie van de useEffect met unlisten in de dep array.

Reply:
@OscarBakker ik snap hier niet welke je bedoelt.

Reply:
of bedoel je de warning mbt handleKeyup?

Reply:
volgens mij heb ik dit nu gefixt